actors/miner/src/types.rs

```diff
@@ -135,6 +135,34 @@ pub struct ProveCommitSectorParams {
     pub proof: RawBytes,
 }
 
+// Note no UnsealedCID because it must be "zero" data.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize_tuple, Deserialize_tuple)]
+pub struct SectorNIActivationInfo {
+    pub sealer_id: Option<ActorID>,
```

Comment:
this is not nullable in the FIP, does this assume https://github.com/filecoin-project/FIPs/pull/993 is accepted?

Reply:
In fact, this isn't even used in this PR; and I see no mention of `SealerID` / `sealer_id` in the text of https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0090.md other than in the description of this struct. The only thing it says about it is _"Must be set to ID of receiving actor for now"_, so I guess it's a placeholder waiting for https://github.com/filecoin-project/FIPs/pull/993? If so, I guess it needs to be checked against the receiving actor in `prove_commit_sectors_ni`.

Reply:
Yes it's a placeholder that must be checked. Shouldn't be optional.

Reply:
Should we fail a whole batch if `sealer_id` of some sector is different that receiving miner ID, or just filter them out?

Reply:
The question is relevant only when not using aggregate proof. I think having aggregate proof for sectors intended for different receiver doesn't make sense.